tezos_interop/tezos_interop.re

```diff
@@ -12,6 +12,7 @@ let rec try_decode_list = (l, string) =>
   };
 module Base58 = Base58;
 module Ed25519 = Ed25519;
```

Comment:
can be removed now that signature is moved to crypto

Reply:
done boss